stdlib/public/core/Tuple.swift.gyb

```diff
@@ -23,6 +23,14 @@ comparableOperators = [
 
 }%
 
+/// Two tuples each of arity 0 are equal
```

Comment:
I would copy the heading from the 'generic' `==` on tuples, and later add a sentence that all empty-tuples are equal to each other as a note.

```swift
// Returns a Boolean value indicating whether the corresponding components of
/// two tuples are equal.
///
/// All zero-arity tuples are equal.
```

Similarly for the `<` etc...